### PR TITLE
Add Purchase.token response

### PR DESCRIPTION
**Pull request by justnero**

### CODE REVIEW

File: openapi/components/schemas/Storefront/Purchase.yaml [hunk 1]
```diff
@@ -20,6 +20,10 @@ properties:
     readOnly: true
     allOf:
       - $ref: StorefrontTransaction.yaml
+  token:
```
Comment (ashkarpetin) [inline openapi/components/schemas/Storefront/Purchase.yaml:23]: Why we need it?

Comment (justnero) [reply openapi/components/schemas/Storefront/Purchase.yaml:23]: For Storefront apps to be able to fetch freshly created resources on the customer behalf.

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/Storefront/Purchase.yaml (hunk 1 above)
Comment (justnero) [reply openapi/components/schemas/Storefront/Purchase.yaml:23]: Sorry, dismissed your approval, just added it in the description to make clearer.

Comment (mattsanf) [reply openapi/components/schemas/Storefront/Purchase.yaml:23]: When a customer follows the transaction's approvalUrl, they are unable to update the details of the transaction after the flow has been followed.

**Review by ashkarpetin — APPROVED** [review]

**Review by mattsanf — APPROVED** [review]

**Pull request merged**